CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Change
+- [vtex abtest] Refactor this commands to use as oclif plugin.
```

Comment:
```suggestion
- [vtex abtest] Refactor to use as oclif plugin.
```

Reply:
I think we have to announce that `vtex workspace abtest` is now `vtex abtest`, right?